milestone.md

```diff
@@ -0,0 +1,199 @@
+3. Wyświetla się pasek postępu ile całkowicie zadań zostało wykonanych
+4. Przycisk "koniec gry", który kończy grę. Wymaga potwierdzenia przez popup. Po zakończeniu przechodzi do ekranu ranking.
+5. Wyświetla się liczba nieocenionych zadań, wraz z przyciskiem "oceniaj", który otwiera ekran [ocenianie](). 
+
+## [GM] Ocenianie 
+1. Co 5 sekund w tle pobierane są nowe odpowiedzi.
+2. Jeżeli wszystkie ocenione odpowiedzi są ocenione, wyświetla się stosowny komunikat.
+3. Jeżeli są nieocenione zadania, wybierane jest to najstarsze.
+4. Wyświetla się nazwa zadania, jego treść oraz przesłane subtaski.
+5. Zadanie można ocenić przy pomocy dwóch przycisków czerwony (odrzucenie) i zielony (akceptacja), bądź poprzez swipe jak na Tinderze.
+6. Po ocenianiu zadania, wyświetlane jest kolejne.
+
+## [GM] Wiadomości
+1. Pole tekstowe do którego można wpisać wiadomość
+2. Przycisk "Wyślij wiadomość do wszystkich"
+
+## [GM] Hamburger menu
+Podczas trwania gry można przemieszczać się między ekranami:
+- gra
+- podgląd zadań
+- odpowiedzi
+- wiadomości
+- ranking
+
+## [Gracz] Lista gier
+1. Wyświetla się lista gier o stanie "oczekiwanie na graczy"
+2. Kliknięcie na grę powoduje przejście do ekranu opis gry
+3. Swipem (przeciągnięciem góra/dół) można odświeżyć istę
+
+## [Gracz] Opis gry
+1. Wyświetla się tytuł gry, opis, data startu (jeżeli istnieje)
+2. Wyświetla się nazwa drużyny, w której obecnie znajduje się gracz
+3. Jeżeli gra jest w stanie "rozgrywka", pokazuje się przycisk "Graj" albo "Przejdź do zadań"
+4. Kliknięcie przycisku powoduje przejście do ekranu [lista zadań]() i uruchomienie w tle procesu wykrywania końca gry.
+
+## [Gracz] Lista zadań
+1. Zadania wyświetlają się w kolejności: z przesłanym błędnym rozwiązaniem, bez przesłanych rozwiązań, zaliczone
+2. Przy każdym zadaniu wyświetla się jego nazwa, liczba punktów i status
+3. Po kliknięciu na zadanie, przejście do ekranu [zadanie]().
+
+## [Gracz] Hamburger menu
+W czasie trwania gry można przechodzić między ekranami:
+- lista zadań
+- ranking
+- wiadomości
+
+## [Gracz] Zadanie
+1. Wyświetla się tytuł zadania, jego treść oraz lista załączników do treści (np. zdjęcie).
+2. Wyświetla się lista odpowiedzi, które użytkownik zgłosił do tego zadania.
+3. Przy każdej odpowiedzi wyświetla się jej status: nieoceniona / zatwierdzona / odrzucona.
+4. Jeżeli nie ma odpowiedzi, bądź żadna nie została zatwierdzona, pokazuje się przycisk "Dodaj odpowiedź", który powoduje przejście do ekranu [udzielania odpowiedzi]().
+
+## [Gracz] Odpowiedź
+1. Wyświetla się tytuł zadania.
+2. Dla każdego subtaska (typu załącznika) pojawia się pole na wprowadzenie jego treści. Jedno pod drugim
+    - TEXT, zwykłe duże pole tekstowe
+    - PHOTO, początkowo wyświetla się ikonka aparatu. Kliknięcie na ikonkę otwiera ekran aparatu. Po poprawnym wykonaniu zdjęcia, ikonka zastępowana jest miniaturką. Zdjęcie można wykonać ponownie poprzez kliknięcie na miniaturkę.
```

Comment:
ładowanie z pliku? ... czy aparat takie coś obsługuje?

Reply:
Myślałem, to akurat jest celowo. Ładowanie z pliku dałoby możliwość wrzucenia jakiejś foci z internetu, a tak to masz w miarę ograniczone, że sam ją wykonujesz.

Reply:
Do przemyślenia później.